scripts/generateDc2InstCat.py

```diff
@@ -0,0 +1,159 @@
+    args = parser.parse_args()
+
+    obshistid_list = args.id
+    opsimdb = args.db
+    out_dir = args.out
+    dither_switch = True
+    if args.dither.lower()[0] == 'f':
+        dither_switch = False
+
+    from lsst.sims.catUtils.utils import ObservationMetaDataGenerator
+
+    if not os.path.exists(opsimdb):
+        raise RuntimeError('%s does not exist' % opsimdb)
+
+    obs_generator = ObservationMetaDataGenerator(database=opsimdb, driver='sqlite')
+
+    from lsst.sims.catUtils.exampleCatalogDefinitions import PhoSimCatalogSersic2D
+    from lsst.sims.catUtils.exampleCatalogDefinitions import PhoSimCatalogZPoint
+    from lsst.sims.catUtils.exampleCatalogDefinitions import DefaultPhoSimHeaderMap
+    from lsst.sims.catUtils.baseCatalogModels import StarObj
+    from lsst.sims.catUtils.baseCatalogModels import GalaxyBulgeObj, GalaxyDiskObj
+    from lsst.sims.catUtils.baseCatalogModels import GalaxyAgnObj
+    from lsst.sims.utils import _getRotSkyPos
+    import copy
+
+    star_db = StarObj(database='LSSTCATSIM', host='fatboy.phys.washington.edu',
+                      port=1433, driver='mssql+pymssql')
+
+    bulge_db = GalaxyBulgeObj(connection=star_db.connection)
+    disk_db = GalaxyDiskObj(connection=star_db.connection)
+    agn_db = GalaxyAgnObj(connection=star_db.connection)
+
+    if not os.path.exists(out_dir):
+        os.mkdir(out_dir)
+
+    phosim_header_map = copy.deepcopy(DefaultPhoSimHeaderMap)
+    phosim_header_map['nsnap'] = 1
+    phosim_header_map['vistime'] = 30.0
+    phosim_header_map['camconfig'] = 1
+    for obshistid in obshistid_list:
+
+        obs_list = obs_generator.getObservationMetaData(obsHistID=obshistid,
+                                                        boundType='circle',
+                                                        boundLength=args.fov)
+
+        obs = obs_list[0]
+        if dither_switch:
+            print 'dithering'
+            obs.pointingRA = np.degrees(obs.OpsimMetaData['randomDitherFieldPerVisitRA'])
+            obs.pointingDec = np.degrees(obs.OpsimMetaData['randomDitherFieldPerVisitDec'])
+            rotSky = _getRotSkyPos(obs._pointingRA, obs._pointingDec, obs,
+                                   obs.OpsimMetaData['ditheredRotTelPos'])
+
+            obs.rotSkyPos = np.degrees(rotSky)
+            obs.OpsimMetaData['rotTelPos'] = obs.OpsimMetaData['ditheredRotTelPos']
+
+        cat_name = os.path.join(out_dir,'phosim_cat_%d.txt' % obshistid)
+        star_name = 'star_cat_%d.txt' % obshistid
+        gal_name = 'gal_cat_%d.txt' % obshistid
+        agn_name = 'agn_cat_%d.txt' % obshistid
+
+        cat = PhoSimCatalogPoint(star_db, obs_metadata=obs)
+        cat.phoSimHeaderMap = phosim_header_map
+        with open(cat_name, 'w') as output:
+            cat.write_header(output)
+            output.write('includeobj %s.gz\n' % star_name)
+            output.write('includeobj %s.gz\n' % gal_name)
+            output.write('includeobj %s.gz\n' % agn_name)
```

Comment:
There is no more AGN catalog to include.  Remove this line (or comment it out as a place holder for when we do add DC2 AGN)